assets/constants.js

```diff
@@ -10,6 +10,7 @@
 var constants = {
   // Whether to allow custom event reporting to Google Analytics.
   "CAN_SEND_ANALYTICS_EVENTS": false,
+  "CAN_SEND_EMAILS": false,
```

Comment:
I found that `CAN_SEND_EMAILS` is also present in `feconf.py`. This will lead to duplication. So, replace all instances of `feconf.CAN_SEND_EMAILS` with `constants.CAN_SEND_EMAILS`.

Reply:
@abhaygupta97 Do I have to remove the line https://github.com/oppia/oppia/blob/develop/core/controllers/profile.py#L258or change all instances of ```feconf.CAN_SEND_EMAILS``` to ```constants.CAN_SEND_EMAILS``` in https://github.com/oppia/oppia/blob/develop/core/controllers/profile.py

Reply:
Yes you have to remove that line. Currently `CAN_SEND_EMAILS` is defined in two places: `feconf.py` and `constants.js`. The definition in `feconf.py` should be removed and wherever `feconf.CAN_SEND_EMAILS` is being used, it should be replaced with `constants.CAN_SEND_EMAILS`

Reply:
But how can I import the constants.js file in the python file (https://github.com/oppia/oppia/blob/develop/core/controllers/profile.py)

Reply:
You can do it using something like [this](https://github.com/oppia/oppia/blob/develop/core/controllers/resources.py#L20).